packages/ai-chat/src/common/chat-service.ts

```diff
@@ -127,6 +130,9 @@ export class ChatServiceImpl implements ChatService {
     @inject(FallbackChatAgentId) @optional()
     protected fallbackChatAgentId: FallbackChatAgentId | undefined;
 
+    @inject(ChatSessionNamingService) @optional()
+    protected chatSessionNamingService: ChatSessionNamingService | undefined;
```

Comment:
We should not directly inject the agent but go through the registry. That way, the user settings regarding enabling/disabling this agent will be taken into account.